src/components/Local.vue

```diff
@@ -175,6 +176,16 @@ export default {
       //  managing the cookies
 
       this.$cookies.set("clips", JSON.stringify(this.clips), Infinity);
+    },
+    prevent_Space(event) {
+      //prevent additional white spaces from adding at start of input
+      if (
+        !this.newClip.length &&
+        (event.keyCode === 13 || event.keyCode === 32)
+      ) {
+        //prevent input if enter or space is pressed when clip is empty
+        event.preventDefault();
+      }
```

Comment:
https://www.instagram.com/p/CARgoKjg1Gp/?utm_source=ig_web_copy_link

Can you try this out and see if it works?

Reply:
- Adding event modifiers to prevent space and enter will prevent them from the whole clip so the user won't be able to add space or enter at any point in the clip. 
- Using v-once will not work because it will prevent space or enter only once.
- In the condition of the logic part, we must put the condition for preventing space and enter in the bracket so that a short circuit won't happen.  [a || b && c will not check b or c condition when a is true so (a || b) && c should be used. ]